src/modules/10-common/strings/strings.en.yaml

```
@@ -1273,3 +1273,5 @@ supportText: Support
 openInNewTab: Open in new tab
 stayOnThisPage: 'Stay on this Page'
 leaveThisPage: 'Leave this Page'
+svcEnv2Banner: 'Harness has made a significant update to the services and environments in CD. Services and Environments now have definitions that are associated with the respective object and can be managed independent of the pipeline. This change is a forward looking change that won’t impact your existing pipelines. However, we do plan on reducing our support on the service v1 and environments v1 and plan to deprecate by end of January. We have an automated migration tool to support our users. Please contact <a href= {{support}} target="_blank" rel="noreferrer"> <b>&nbsp;support.harness.io&nbsp;</b> </a> for further questions.'
+svcEnvV2Truncated: '  Harness has made a significant update to the services and environments in CD. Services and Environments now have definitions that are associated with the respective object and can be managed ...'
```

space at the beginning
```suggestion
svcEnvV2Truncated: 'Harness has made a significant update to the services and environments in CD. Services and Environments now have definitions that are associated with the respective object and can be managed ...'
```

---

done